### ISSUE TITLE
执行.to(DEVICE)或.cuda()卡住

### ISSUE THREAD

**Issue by PYNing** [Oneflow-Inc/oneflow#9595]
**问题描述：**
通过 **pip install oneflow==0.8.0** 安装的版本，执行.to(DEVICE)或.cuda()会卡住(等了10分钟以上)，一个CPU核跑满，无warning也不报错。
通过 **python3 -m pip install -f https://release.oneflow.info oneflow==0.8.0+cu112** 安装的版本则正常

**环境：**
GPU A100*1，CUDA 11.2，驱动 450.80.02，python 3.10，conda环境

**复现代码：**
`
from oneflow import nn

class NeuralNetwork(nn.Module):
    def __init__(self):
        super(NeuralNetwork, self).__init__()
        self.flatten = nn.Flatten()
        self.linear_relu_stack = nn.Sequential(
            nn.Linear(28*28, 512),
            nn.ReLU(),
            nn.Linear(512, 512),
            nn.ReLU(),
            nn.Linear(512, 10),
            nn.ReLU()
        )

    def forward(self, x):
        x = self.flatten(x)
        logits = self.linear_relu_stack(x)
        return logits
        
net = NeuralNetwork()
print(net)
net = net.cuda()
print("done")
`
日志：
<img width="289" alt="微信图片_20221213103448" src="https://user-images.githubusercontent.com/11667149/207216493-616e5545-d6c7-489c-9fd8-1ee2219652d3.png">

**Comment by shangguanshiyuan** [comment]
您好，pypi上上传的oneflow是cuda 10.2版本的，a100不支持cuda 10.2，在a100上运行cu102的安装包会触发jit出现像是无响应的现象，您在a100上请使用cuda 11版本的安装包，版本后缀为cu112。

**Comment by PYNing** [comment]
感谢！
